tests/docs/test_docs_training_data.py

```diff
@@ -24,6 +24,13 @@
 
 @pytest.mark.parametrize("mdx_file_path", MDX_DOCS_FILES)
 def test_docs_training_data(mdx_file_path: Path):
+    if mdx_file_path == Path("docs/docs/migration-guide.mdx"):
```

Comment:
@m-vdb That's the part which'd require review by you

Reply:
a cleaner way could be to remove the path from the `MDX_DOCS_FILES` variable with `MDX_DOCS_FILES.remove(Path("docs/docs/migration-guide.mdx"))` after line 16.